### PR TITLE
Styling auth

### PR DESCRIPTION
**Pull request by VisahanA**

### CODE REVIEW

File: src/next/components/auth/AuthHeader.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export function AuthHeader() {
+  //
+  // onSubmit = () => {
+  //   if(userFound){
+  //     this.props.history.push('/posts/');
+  //   }
+  // }
+  return (
+    <>
+      <div className={`${styles['auth-header']} ${styles['testing1']}`}>
```
Comment (skavinvarnan) [inline src/next/components/auth/AuthHeader.js:12]: Refactor the css name from testing1

File: src/next/components/auth/AuthMain.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import styles from './AuthMain.module.scss'
```
Comment (skavinvarnan) [inline src/next/components/auth/AuthMain.js:1]: Remove AuthMain completely

File: src/next/sass/global.scss [hunk 3]
```diff
@@ -1,4 +1,11 @@
+@import "./init";
+
+body {
+  height: 100%;
+  cursor: default;
+  margin: 0;
+}
+
 .testing {
```
Comment (skavinvarnan) [inline src/next/sass/global.scss:9]: Removing testing

File: src/pages/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import SignUpPage from '../next/pages/SignUpPage';
```
Comment (skavinvarnan) [inline src/pages/SignUp.js:1]: Remove SignUp page

Comment (VisahanA) [reply src/pages/SignUp.js:1]: Resolved

File: src/next/sass/global.scss (hunk 3 above)
Comment (VisahanA) [reply src/next/sass/global.scss:9]: Resolved

File: src/next/components/auth/AuthMain.js (hunk 2 above)
Comment (VisahanA) [reply src/next/components/auth/AuthMain.js:1]: Resolved the issue

File: src/next/components/auth/AuthHeader.js (hunk 1 above)
Comment (VisahanA) [reply src/next/components/auth/AuthHeader.js:12]: Removed testing1 since it is not in use

**Comment by skavinvarnan** [comment]
@VisahanA The design base structure looks good.
Have two thoughts though
1) It would be great if page is not scrollable
2) Could be add more "Box Value" its in a react loop, so we can visualise how the scrolling feels

**Pull request merged**